libraries/exoplayer/src/main/java/androidx/media3/exoplayer/drm/DummyExoMediaDrm.java

```diff
@@ -156,4 +156,14 @@ public CryptoConfig createCryptoConfig(byte[] sessionId) {
   public @C.CryptoType int getCryptoType() {
     return C.CRYPTO_TYPE_UNSUPPORTED;
   }
+
+  @Override
+  public void removeOfflineLicense(byte[] keySetId) {
+    // Do nothing
```

Comment:
For consistency with methods above that take `keySetId` (like `getKeyRequest`) this should probably throw an exception

Reply:
Thanks for reviewing. Updated.